modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,173 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, {SMProductName} (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like {product-title} monitoring for user-defined projects.
+
+The following steps show how to integrate Service Mesh with user-workload monitoring.
+
+.Prerequisites
+
+* You have cluster administrator permissions.
+
+.Procedure
+
+. Enable monitoring in user-defined projects by adding the following configuration to the `cluster-monitoring-config` config map:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+----
+
+. Configure the SMCP for user-workload monitoring:
```

Comment:
Actually, this SMCP config has nothing specific for user-workload monitoring. Any Prometheus instance could monitor this SMCP instance.
```suggestion
. Configure the SMCP for external Prometheus:
```